benchmarks/test_cli.py

```diff
@@ -7,42 +7,60 @@
 from integration.util import cli
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
I've always been a little confused about "autouse" and why you'd want it.

Does this mean: "in (only) this `test_cli` module, please assume every test had `cli_alias` fixture declared"...?

Reply:
(Yes).